src/isPalindrome.js

```diff
@@ -20,6 +20,20 @@
  */
 function isPalindrome(str) {
   // write code here
+  const string = str.toLowerCase();
+
+  const symbols = /[^A-Za-z0-9]/gi;
```

Comment:
это шаблон для регулярного выражения
```suggestion
  const pattern = /[^A-Za-z0-9]/gi;
```